packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md

```diff
@@ -30,21 +30,21 @@ Okta defines several different types of inline hooks. Each type of inline hook m
 | [SAML assertion inline hook](/docs/reference/saml-hook/)       | Customizes SAML assertions returned by Okta                                    |
 | [Registration inline hook](/docs/reference/registration-hook/) | Customizes handling of Self-Service Registration (SSR) and Progressive Enrollment support |
 | [Password import inline hook](/docs/reference/password-hook/)  | Verifies a user-supplied password to support migration of users to Okta        |
-| [Telephony inline hook](/docs/reference/telephony-hook/) | Customizes Okta's flows that send SMS or Voice messages |
+| [Telephony inline hook](/docs/reference/telephony-hook/) | Customizes Okta's flows that send SMS or voice messages |
 
 ## Inline hook process flow
 
 ### Extension points
 
-The points in Okta process flows where inline hooks can be triggered are called extension points, because they are where you can extend Okta functionality. Each type of inline hook is triggered at a particular extension point in a particular Okta process flow. At an extension point, if you have configured an inline hook, Okta calls your external service, and waits for a response. When the response is received, Okta resumes the process flow.
+The points in Okta process flows where inline hooks can be triggered are called extension points, because they’re where you can extend Okta functionality. Each type of inline hook is triggered at a particular extension point in a particular Okta process flow. At an extension point, if you’ve configured an inline hook, Okta calls your external service, and waits for a response. When the response is received, Okta resumes the process flow.
```

Comment:
It may not matter, but you might want to disable directional quotes in your editor/Mac, the one used in "they're" and "you've" is directional

Reply:
Thanks! Fixed that up with your extension suggestion.